src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -636,11 +636,11 @@ namespace smt::noodler {
         for (const auto& we : m_word_diseq_todo) {
             app_ref eq(m.mk_eq(we.first, we.second), m);
             app_ref dis(m.mk_not(eq), m);
-            // if(!ctx.is_relevant(dis.get())) {
-            //     STRACE("str", tout << "remove_irrelevant NEQ: " << mk_pp(dis.get(), m) << " relevant: " <<
-            //         ctx.is_relevant(dis.get()) << " assign: " << ctx.find_assignment(dis.get()) << '\n';);
-            //     continue;
-            // }
```

Comment:
Is enabling these previously commented out lines intentional? Same goes for all the other enabled previously commented out chunks.